homeassistant/components/roku/icons.json

```diff
@@ -0,0 +1,37 @@
+{
+  "entity": {
+    "binary_sensor": {
+      "headphones_connected": {
+        "default": "mdi:headphones"
+      },
+      "supports_airplay": {
+        "default": "mdi:cast-variant"
+      },
+      "supports_ethernet": {
+        "default": "mdi:ethernet"
+      },
+      "supports_find_remote": {
+        "default": "mdi:remote"
+      }
+    },
+    "select": {
+      "application": {
+        "default": "mdi:application"
+      },
+      "channel": {
+        "default": "mdi:television"
+      }
+    },
+    "sensor": {
+      "active_app": {
+        "default": "mdi:application"
+      },
+      "active_app_id": {
+        "default": "mdi:application-cog"
+      }
+    }
```

Comment:
It looks like `sensor.py` (with removal of these icons) wasn't committed.
`test_sensor.py` also needs to be changed then (to not check for the icons).